src/cmd/usb-util.js

```diff
@@ -139,36 +141,42 @@ async function executeWithUsbDevice({ args, func, dfuMode = false } = {}) {
 			await device.close();
 		}
 	}
-	return res;
 }
 
 /**
- * Reboots device and waits for it to enter normal mode.
+ * Waits for device readiness (mainly to send control requsts to it)
  * Useful for enabling Device Protection on a device in after its current operation completes.
  * @param {*} deviceId
  * @returns
  */
-async function waitForDeviceToRespond(device) {
+async function waitForDeviceToRespond(device, { timeout = 10000 } = {}) {
 	const deviceId = device.id;
-	const REBOOT_TIME_MSEC = 20000;
-	const REBOOT_INTERVAL_MSEC = 1000;
+	const REBOOT_TIME_MSEC = timeout;
+	const REBOOT_INTERVAL_MSEC = 100;
```

Comment:
With this interval reduced, let's pay attention during testing that `waitForDeviceToRespond` doesn't return too early with the old device. For example, if the device got the command to reboot, but hasn't done so after the 100 ms delay, could this return early? I'm not sure if that's a possibility, just pointing out something to keep an eye for. (no change to the code requested here).